### PR TITLE
Expose the number of clients that are connected to XMPP and MUCs.

### PR DESCRIPTION
**Pull request by bgrozev**

### CODE REVIEW

File: jicoco/src/main/java/org/jitsi/xmpp/mucclient/MucClientManager.java [hunk 1]
```diff
@@ -341,4 +341,34 @@ public boolean removeMucClient(String id)
...
+        return mucClients.values().stream()
+                .filter(MucClient::isConnected)
+                .count();
+    }
+
+    /**
+     * Return the number of {@link MucClient}s that are succesfully connected
+     * to XMPP and joined all of their configured MUCs.
+     */
+    public long getJoinedClientCount()
```
Comment (bbaldino) [inline jicoco/src/main/java/org/jitsi/xmpp/mucclient/MucClientManager.java]: Should we track the actual counts (number of actual connections vs number of configured/expected connections) vs 'how many clients connected to everything they were configured to'?

Comment (bgrozev) [reply jicoco/src/main/java/org/jitsi/xmpp/mucclient/MucClientManager.java]: This makes it hard to interpret the graphs -- how many joined MUCs do we expect?

If we have more than one MUC configured for an XMPP server, and we only joined some of them, this is enough to signal that something is wrong

Comment (bbaldino) [reply jicoco/src/main/java/org/jitsi/xmpp/mucclient/MucClientManager.java]: We have the total number of expected, no?  `mucs.size()` in `MucClient`?

Comment (bgrozev) [reply jicoco/src/main/java/org/jitsi/xmpp/mucclient/MucClientManager.java]: True. Then we have to deal with the difference in the number of expected connecte clients vs the number of mucs joined

**Pull request merged**